core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -849,11 +856,11 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 
 			foreach ($arrDirections as $dir)
 			{
-				$label = $GLOBALS['TL_LANG'][$strTable][$dir][0] ?: $dir;
-				$title = $GLOBALS['TL_LANG'][$strTable][$dir][1] ?: $dir;
+				$label = $GLOBALS['TL_LANG'][$strTable][$dir][0] ?? $dir;
+				$title = $GLOBALS['TL_LANG'][$strTable][$dir][1] ?? $dir;
 
 				$label = Image::getHtml($dir . '.svg', $label);
-				$href = $v['href'] ?: '&amp;act=move';
+				$href = $v['href'] ?? '&amp;act=move';
```

Comment:
These three lines behave different now if the value is the empty string for example, is this intended?

Reply:
No. Fixed in 1550fd769ce350112719147253a37736f5c7cd79.